<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局样式 */
QWidget {
    background-color: #FFFFFF;
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 20px;
}

/* Logo 样式 */
QLabel#LogoLabel {
    font-size: 20px;
    font-weight: bold;
    color: #FF6A00;
    border: 2px solid #FF6A00;
    border-radius: 10px;
    background-color: #FFF2E6;
    padding: 10px;
    text-align: center;
}

/* 标题样式 */
QLabel#WelcomeLabel {
    font-size: 20px;
    font-weight: bold;
    color: #333333;
}

/* 表单标签样式 */
QLabel {
    font-size: 20px;
    color: #333333;
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #CCCCCC;
    border-radius: 5px;
    padding: 5px;
    font-size: 20px;
    background-color: #FAFAFA;
}

QLineEdit:focus {
    border: 1px solid #FF6A00;
}

/* 按钮样式 */
QPushButton {
    background-color: #FF6A00;
    color: white;
    border-radius: 5px;
    padding: 8px;
    font-size: 20px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #FF4500;
}

QPushButton:pressed {
    background-color: #CC3700;
}

/* 用户协议复选框 */
QRadioButton#AgreementRadioButton {
    font-size: 20px;
    color: #555555;
}

QRadioButton::indicator {
    width: 16px;
    height: 16px;
}

QRadioButton::indicator:checked {
    background-color: #FF6A00;
    border: 1px solid #FF6A00;
}
</string>
  </property>
  <widget class="QLabel" name="LogoLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="GoBack">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="WelcomeLabel">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>90</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>欢迎注册</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="FormWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>329</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="FormLayout">
    <item row="3" column="1">
     <widget class="QLineEdit" name="PhoneLineEdit">
      <property name="placeholderText">
       <string>请输入手机号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="NicknameLabel">
      <property name="text">
       <string>昵称</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="NicknameLineEdit">
      <property name="placeholderText">
       <string>请输入昵称</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="PasswordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>请输入密码</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="PhoneLabel">
      <property name="text">
       <string>手机号</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="PasswordLabel">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="ComfirmPwdLabel">
      <property name="text">
       <string>确认密码</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="ConfirmPwdLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>再次输入密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="AgreementRadioButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>我已阅读并同意用户协议</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>立即注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="VersionLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>版本号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
